mm2src/coins/lp_coins.rs

```diff
@@ -4813,8 +4851,16 @@ pub async fn send_raw_transaction(ctx: MmArc, req: Json) -> Result<Response<Vec<
         Ok(None) => return ERR!("No such coin: {}", ticker),
         Err(err) => return ERR!("!lp_coinfind({}): {}", ticker, err),
     };
-    let bytes_string = try_s!(req["tx_hex"].as_str().ok_or("No 'tx_hex' field"));
-    let res = try_s!(coin.send_raw_tx(bytes_string).compat().await);
+    // tx_json parsing is required for siacoin because txes are never encoded in hex
+    let tx_string = if let Some(tx_hex) = req["tx_hex"].as_str() {
+        tx_hex.to_owned()
+    } else if let Some(tx_json) = req["tx_json"].as_object() {
+        let json_string = try_s!(json::to_string(tx_json));
+        json_string
+    } else {
+        return ERR!("No 'tx_hex' or 'tx_json' field");
+    };
+    let res = try_s!(coin.send_raw_tx(&tx_string).compat().await);
```

Comment:
Maybe send_raw_tx now should accept the param as `enum { TxHex(String), TxJson(Value) }`